README.md

```diff
@@ -23,6 +23,60 @@ config :my_app, MyAppWeb.Endpoint,
 
 The default interval is 100ms.
 
+
+## Streaming serving logs to the web console
+
+Streaming server logs that you see in the terminal when running `mix phx.server` can be useful to have on the client during development, especially when debugging with SPA fetch callbacks, GraphQL queries, or LiveView actions in the browsers web console. You can enable log streaming to collocate client and server logs in the web console with the `web_console_logger` configuration in your `config/dev.exs`:
+
+```elixir
+config :my_app, MyAppWeb.Endpoint,
+  live_reload: [
+    interval: 1000,
+    patterns: [...],
+    web_console_logger: true
+  ]
+```
+
+Next, you'll need to listen for the `"phx:live_reload:loaded"` event and enable client logging by calling the reloader's `enableServerLogs()` function, for example:
+
+```javascript
+window.addEventListener("phx:live_reload:loaded", ({detail: {reloader}}) => {
```

Comment:
Can we get rid of one level of matching? Or is this pretty much required by JS?

```suggestion
window.addEventListener("phx:live_reload:loaded", ({detail: reloader}) => {
```

Reply:
Maybe this even should be called "connected"? `phx:live_reload:connected`? I think we should only emit it once we connect, right?